src/ontology/mondo-edit.obo

```diff
@@ -579746,6 +579746,15 @@ property_value: http://purl.org/dc/elements/1.1/date "2024-07-18T18:06:40Z" xsd:
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7858" xsd:anyURI
 
+[Term]
+id: MONDO:1040020
+name: methicillin-susceptible staphylococcus aureus infectious disease
+def: "A bacterial disease caused by methicillin-susceptible staphylococcus aureus." [https://orcid.org/0000-0002-0587-4693, MONDO:patterns/infectious_disease_by_agent, PMID:33144192]
```

Comment:
@yshwetar please update the beginning to "a bacterial **infectious** disease".
Otherwise, the rest is great.

Reply:
done

Reply:
Looks like I'm too slow to respond! Thanks Sabrina :)

Reply:
I was just there, I thought I would just do it :-) nothing to do with your speed :-)